mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -422,21 +422,27 @@ uploadWithMode m f a = do
       -- better default.
       case s > 1024 * 1024 * 1024 of
         True ->
-          multipartUpload f a s (2 * chunk) 20
+          multipartUpload f a s (2 * bigChunkSize) 20
         False ->
-          multipartUpload f a s chunk 20
+          multipartUpload f a s bigChunkSize 20
+
+
+
+bigChunkSize :: Integer
+bigChunkSize = 100 * 1024 * 1024
```

Comment:
What's the reasoning behind this number?

Reply:
Just extracted from the above function

Reply:
So it could be used elsewhere.